### PR TITLE
Assignment4 @ZackWard-student

### PR DESCRIPTION
**Pull request by sammyboy2000**
Web-app: https://icy-stone-0b3d2a71e.1.azurestaticapps.net/dailyGame
API: https://inconceivable-api.azurewebsites.net/swagger/index.html

### CODE REVIEW

**Comment by linhtran8699** [comment]
I don't think I am able to click on anything from the hamburger menu. 
Also, that's a fun background.

**Review by mpennock5 — COMMENTED** [review]
Your theme is awesome!! It looks like the feedback for the letters and their position is not currently functioning for the daily game. I am unable to use your nav menu on the top right. 
I like that you added the word list to the client so you are not making a bunch of api calls to update the available words. 
I was not able to test the last 10 daily game stats. Did not appear to have any data seeded to the database to demo that part of the dailyGameStats page but the method the controller calls in the service looked good from what I could tell without testing it.

Looking good so far though!

**Comment by hewrtarkhany** [comment]
cool video effect,

File: Wordle.Api/Dtos/DateWordDto.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+namespace Wordle.Api.Dtos
+{
+    public class DateWordDto
+    {
+        public DateTime Date { get; set; }
+        public int NumPlays { get; set; }
+        public int AverageScore { get; set; }
+        public int AverageTime { get; set; }
+        public bool HasPlayed { get; set; }
```
Comment (mmwoodfo) [inline Wordle.Api/Dtos/DateWordDto.cs:9]: This doesn't seem to be getting set.

**Pull request closed without merging**